src/Animations/babylon.animation.ts

```diff
@@ -529,14 +529,27 @@
 
             for (index = 0; index < parsedAnimation.keys.length; index++) {
                 var key = parsedAnimation.keys[index];
-
+                var inTangent:any;
+                var outTangent:any;
 
                 switch (dataType) {
                     case Animation.ANIMATIONTYPE_FLOAT:
                         data = key.values[0];
+                        if (key.values.length >= 1) {

```

Comment:
this is not mandatory. What if the content was designed to be linear?

Reply:
thanks your feedback!
As you said, if the animation was linear interpolation, we don't have to store tangent value.

But according to my observation, If the length of key.values is higher than one, even though the value is zero, it have to store inTanget and outTangent value for reproducing the purpose of animation.

## linear interpolation
![linear_interpolation](https://user-images.githubusercontent.com/4597574/31598253-841df63a-b287-11e7-88a1-a4f44dedc95b.png)

## tangent interpolation
![tanget_interpolation](https://user-images.githubusercontent.com/4597574/31598261-8b52cf84-b287-11e7-81da-6f7f0bd8e670.png)

Reply:
I can emit an animation with 100 keyframes and still wanting it to be linear